### PR TITLE
Update README.md with process description improvements.

### PR DESCRIPTION
**Pull request by hansfn**
Update name of Ideas issue.

Also tried to make it clearer where issues should be created - in the sandbox. Or did "there" refer to https://www.drupal.org/project/dashboard

### CODE REVIEW

**Comment by penyaskito** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr23-y8mtwpxp4humrv7lojyuvzbixxlkzmze.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64c8ec61f70224ca0961ad38

File: README.md [hunk 1]
```diff
@@ -6,12 +6,13 @@
...
+Create a new and modern Dashboard based on user needs, and define those needs per role.
 
-The homepage of the initiative is at the Ideas issue queue. See ["Modern Dashboard with Role presets"](https://www.drupal.org/project/ideas/issues/3244581), which provides a lot of context.
+The homepage of the initiative is at the Ideas issue queue. See ["Enhance user experience with customizable dashboards"](https://www.drupal.org/project/ideas/issues/3244581)
+(was "Modern Dashboard with Role presets") on drupal.org, which provides a lot of context.
 
-Development happens on a [sandbox](https://www.drupal.org/sandbox/penyaskito/3327580), as we are still working on getting access to
-the [Dashboard namespace](https://www.drupal.org/project/dashboard). Feel free to create issues there, as we will move them when we are able to.
+Development happens in a [sandbox project](https://www.drupal.org/sandbox/penyaskito/3327580), as we are still working on getting access to
+the [Dashboard namespace](https://www.drupal.org/project/dashboard). Feel free to create issues in the sandbox, as we will move them when we are able to.
```
Comment (penyaskito) [inline README.md:15]:
```suggestion
Development happens in a [sandbox project](https://www.drupal.org/sandbox/penyaskito/3327580). Feel free to create or work on issues in the sandbox.
```

We have access to the dashboard namespace now. We decided to not use it.

**Review by penyaskito — APPROVED** [review]
Thanks for the PR!

**Comment by penyaskito** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr23-y8mtwpxp4humrv7lojyuvzbixxlkzmze.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64c8ec61f70224ca0961ad38

File: README.md [hunk 2]
```diff
@@ -6,12 +6,12 @@
...
-Create a new and modern Dashboard based on user needs, and define that needs per role.
+Create a new and modern Dashboard based on user needs, and define those needs per role.
 
-The homepage of the initiative is at the Ideas issue queue. See ["Modern Dashboard with Role presets"](https://www.drupal.org/project/ideas/issues/3244581), which provides a lot of context.
+The homepage of the initiative is at the Ideas issue queue. See ["Enhance user experience with customizable dashboards"](https://www.drupal.org/project/ideas/issues/3244581)
+(was "Modern Dashboard with Role presets") on drupal.org, which provides a lot of context.
 
-Development happens on a [sandbox](https://www.drupal.org/sandbox/penyaskito/3327580), as we are still working on getting access to
-the [Dashboard namespace](https://www.drupal.org/project/dashboard). Feel free to create issues there, as we will move them when we are able to.
+Development happens in a [sandbox project](https://www.drupal.org/sandbox/penyaskito/3327580). Feel free to create or work on issues in the sandbox.
```
Comment (penyaskito) [inline README.md:14]:
```suggestion
Development happens in a [sandbox project](https://www.drupal.org/sandbox/penyaskito/3327580). Feel free to create or work on issues in the sandbox.

If you can, also create a PR here applying the patch via composer.json. This helps with:
* Running the tests, as sandboxes don't have CI enabled.
* Get a tugboat link with a site with the patch applied where people can test the fix/improvement.
* Giving appropiate credit here too for your contribution!
```

Comment (penyaskito) [reply README.md:14]: Improved the description of the process about how we are using github.

**Review by penyaskito — APPROVED** [review]

**Comment by penyaskito** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr23-y8mtwpxp4humrv7lojyuvzbixxlkzmze.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64c8ec61f70224ca0961ad38

**Comment by penyaskito** [comment]
@hansfn Did some tweaks myself. Please confirm you are happy with them and I'll merge this. Thanks!

**Comment by penyaskito** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr23-y8mtwpxp4humrv7lojyuvzbixxlkzmze.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64c8ec61f70224ca0961ad38

**Comment by penyaskito** [comment]
@hansfn Oh right! Can you move it?

**Comment by penyaskito** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr23-y8mtwpxp4humrv7lojyuvzbixxlkzmze.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64c8ec61f70224ca0961ad38

**Comment by hansfn** [comment]
> @hansfn Oh right! Can you move it?

Done. As you probably already noticed.

**Comment by penyaskito** [comment]
@hansfn Thanks!

I think this could use some more improvement, but definitely better than what we had.

**Pull request merged**